app/containers/AudioPlayer/PlaybackSpeed.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Text, TouchableOpacity } from 'react-native';
+
+import styles from './styles';
+import { useTheme } from '../../theme';
+import { AVAILABLE_SPEEDS } from './constants';
+
+const PlaybackSpeed = ({
```

Comment:
To review with design: changing this value in one audio is supposed to change on all audios?
Should it have an MMKV item for that?

Reply:
I talked with Jose and he thinks it's a great idea to do this approach